ci/README.md

```diff
@@ -0,0 +1,111 @@
+# service-telemetry-operator CI playbooks
+
+The playbooks in this directory are used by zuul jobs, which are defined in ../.zuul.yaml.
+
+## Job descriptions
+
+### PR jobs
+
+There are 6 jobs run on every PR that is targeting master.
+These are reported under the `rdoproject.org/github-check` check.
+
+Two scenarios run:
+- `local_build`, which replaces build the STF images and deploys by creating a STF object.
+- `local_build-index_deploy`, which builds the images and does an index-based deployment
+
+Each of these scenarios run across the following OCP versions:
+- 4.12
+- 4.13
+- 4.14
+
+### Periodic jobs
```

Comment:
It will be good to mention that periodic jobs is being run daily.

Reply:
This is mentioned as "jobs are run nightly"